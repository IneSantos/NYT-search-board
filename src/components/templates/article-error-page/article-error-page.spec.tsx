import React from "react";
import { render } from "@testing-library/react";
import ArticleErrorPage from "./article-error-page";

describe("ArticleErrorPage component tests", () => {
  test(`ArticleErrorPage component render`, () => {
    const { container } = render(<ArticleErrorPage />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
