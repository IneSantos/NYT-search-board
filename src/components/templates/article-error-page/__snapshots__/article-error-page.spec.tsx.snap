// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ArticleErrorPage component tests ArticleErrorPage component render 1`] = `
<section>
  <header>
    <div
      class="banner"
    >
      “The New York Times” article search application
    </div>
  </header>
  <div
    class="page-body-wrapper"
  >
    <div
      class="page-body"
    >
      <div
        class="article-error"
      >
        <a
          class="article-action"
          href="/"
        >
           
          &lt; Go to results page
           
        </a>
        <div
          class="description"
        >
          Invalid URL please go back to the Results Page
        </div>
      </div>
    </div>
  </div>
</section>
`;
